lib/common_test/src/test_server_ctrl.erl

```diff
@@ -1152,11 +1153,13 @@ init_tester(Mod, Func, Args, Dir, Name, {_,_,MinLev}=Levels,
 	end,
     OkN = get(test_server_ok),
     FailedN = get(test_server_failed),
+	TotalTestTime = get(test_server_total_time),
     print(html,"\n</tbody>\n<tfoot>\n"
 	  "<tr><td></td><td><b>TOTAL</b></td><td></td><td></td><td></td>"
-	  "<td>~.3fs</td><td><b>~ts</b></td><td>~w Ok, ~w Failed~ts of ~w</td></tr>\n"
+	  "<td>~.3fs<br></td><td><b>~ts</b></td><td>~w Ok, ~w Failed~ts of ~w<br>
+	   Total time: ~.3fs</td></tr>\n"
```

Comment:
@Whaileee, @Mikaka27  - let's have a discussion about naming, decide on some convention and then update internal notes https://github.com/Mikaka27/otp/blob/michal/common_test/time_in_index/OTP-18981/lib/common_test/internal_doc/ct_notes.md

TotalTestTime in variable name and `Total time` displayed in logs is very similar and maybe could be improved ...